companies/management/commands/sync_corp_codes.py

```
@@ -89,6 +176,29 @@ def handle(self, *args, **options):
                 )
             )
 
+            # 기업 필터링 (--top-companies 또는 --limit 옵션)
+            original_count = len(companies_data)
```

Ruff (0.14.11)

Local variable `original_count` is assigned to but never used

Remove assignment to unused variable `original_count`

(F841)